dbt/models/default/schema/default.vw_pin_value.yml

```diff
@@ -43,8 +49,48 @@ models:
         description: '{{ doc("shared_column_year") }}'
 
     tests:
+      - not_accepted_values:
+          name: default_vw_pin_value_mailed_class_no_hyphens
+          column_name: mailed_class
+          values: "2-99"
```

Comment:
I'm not entirely sure if wrapping this in double quotes is correct. The test ran without erring with no quotes as well...

Reply:
If there's a dash in it then it's going to be converted into a string anyways, so this should be fine.